tests/integration/test_envelope.py

```diff
@@ -200,7 +200,6 @@ def test_normalize_measurement_interface(
         "inp": {"unit": "millisecond", "value": 100.14},
         "fp": {"unit": "millisecond", "value": None},
         "lcp": {"unit": "millisecond", "value": 420.69},
-        "missing_value": None,
```

Comment:
Previously, this test would send an event through without a MeasuremntConfig, which would result in `fn remove_invalid_measurements` not being called, but now that we will get a measurementconfig from globalconfig, this function is called, and this line is dropped completely. To me this seems fine but please have a look that it's fine for me to edit the expected test output here.

Reply:
Where was the `missing_value` entry added?

Reply:
here: https://github.com/getsentry/relay/pull/2415/files/4cda1ea5f666d7ffe285863cee15a2958e856984#diff-12f50ce86ac73bfd9ac75e584534e50a3e3da9011a4f63ca9ee96740f1ed69b6R184

Reply:
ah, this link should work:  https://github.com/getsentry/relay/blob/master/tests/integration/test_envelope.py#L184

as you can see it's just something they sent within the same test, so the value is kept when theres no measurementsconfig available, but now that we fetch from global, it will be removed due to this condition: https://github.com/getsentry/relay/blob/master/relay-event-normalization/src/normalize/mod.rs#L456